### ISSUE TITLE
Cider implementation for for super-initializer parameters

### ISSUE THREAD

**Issue by devoncarew** [dart-lang/sdk#48070]
See https://github.com/dart-lang/sdk/issues/48055 for the main tracking issue.

**Comment by devoncarew** [comment]
cc @keertip

**Comment by stereotype441** [comment]
Note: in our planning meeting on Dec 13, it was suggested that when the user hovers over an optional super parameter that inherits a default value from the corresponding parameter on the base class, the IDE should show the default value that's being inherited from the base class constructor.  Doing so requires work in the analysis server, and possibly in some of the IDE integration logic.

**Comment by leafpetersen** [comment]
FYI, the flag has been flipped so these can be used without a flag internally now.

**Comment by itsjustkevin** [comment]
@leafpetersen per your comment, this issue seems like it can be closed.

**Comment by leafpetersen** [comment]
No, my comment was just to indicate that work on this can start.

**Comment by leafpetersen** [comment]
@keertip do you have a status on this?
